test/directives003/expected

```diff
@@ -0,0 +1,6 @@
+nuyahcha
+mohgiide
+apeekair
+eeraimam
+rahhahwi
+goocheiv
```

Comment:
Are these just random words or do they mean something in some language?

Reply:
They are random words generated with `pwgen -AB0`.